src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
```

Comment:
I think that the card should be in the div tag, since the section tag is intended for dividing the page into meaningful blocks, somehow

Reply:
I think that all elements of the product card are interrelated, so the section tag can be used here.

Reply:
eh, don't know about it
i'd use div, a single product card is surely not a section of the page

Reply:
section change to div